src/CoreWCF.Primitives/src/CoreWCF/Channels/BufferedMessageData.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Buffers;
 using System.Xml;
```

Comment:
You've added this using but there's not code change.

Reply:
done thanks